hca/auth/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+Authorization and Authentication system
+***************************************
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ..util import SwaggerClient
+
+
+class AuthClient(SwaggerClient):
```

Comment:
This is just a stub for now I'm guessing?

Reply:
No, this is the API client for Fusillade. It doesn't need any extra code here.

Reply:
no this generates the python api, and cli bindings